__tests__/reducers/literal.test.js

```diff
@@ -1,88 +1,136 @@
-import { literal } from '../../src/reducers/literal'
+import { removeAllContent, removeMyItem, setMyItems  } from '../../src/reducers/literal'
 
 describe('literal reducer', () => {
-  it('should handle initial state', () => {
-    expect(
-      literal(undefined, {})
-    ).toEqual({formData: []})
-  })
 
   it('should handle SET_ITEMS', () => {
-    expect(
-      literal({formData: []}, {
-        type: 'SET_ITEMS',
-        payload: {id: 1, uri:'Run the tests', items: []}
-      })
-    ).toEqual({
-      "formData": [{
-        "id": 1, "uri": "Run the tests", "items": []
-      }]
+    const literalSetItems = setMyItems({ "resourceTemplate:Monograph:Instance": {
+      'http://schema.org/name': { items: [] }
+    }}, {
+      type: 'SET_ITEMS',
+      payload: {
+        rtId: 'resourceTemplate:Monograph:Instance',
+        uri:'http://schema.org/name',
+        items: [ { id: 0, content: 'Run the tests'} ]
+      }
+    })
+    expect(literalSetItems).toEqual({
```

Comment:
this expect seems to be about "replaces empty items with the payload items"

Reply:
The description should now make it clear that the test is about changing the Redux state based on the SET_ITEMS action being passed to the reducer (this should be the most common use case for the reducer).